public/docs/ts/latest/guide/user-input.jade

```diff
@@ -150,76 +148,67 @@ figure.image-display
 :marked
   ## On blur
 
-  Our previous example won't transfer the current state of the input box if the user mouses away and clicks
-  elsewhere on the page. We update the component's `values` property only when the user presses Enter
+  If the user mouses away and clicks elsewhere on the page,
+  the previous example won't transfer the current state of the input box.
+  You update the component's `values` property only when the user presses Enter
```

Comment:
Let's rewrite this to avoid "You" and be more accurate. Maybe:

The component's `values` property updates only when...